tests/beaker_tests/lib/utilitylib.rb

```diff
@@ -339,13 +345,13 @@ def test_manifest(tests, id)
         # if !code.include?($?.exitstatus)
         #  raise 'Errored test'
         # end
-        output = `#{AGENTLESS_COMMAND} --apply #{@temp_agentless_manifest.path}`
+        output = `#{AGENTLESS_COMMAND} --apply #{@temp_agentless_manifest.path} 2>&1`
         unless output.include? 'Applied catalog'
```

Comment:
This seems to be causing problems and the tests are failing for the `multiple_auths` tests as the output doe not include `'Applied catalog'`

Reply:
I may have missed something from the rebase merge, let me check

Reply:
![image](https://user-images.githubusercontent.com/29575714/48612893-258fc700-e982-11e8-9d1f-08158535f525.png)

Yep merge gone bad

I'll fix that up